components/rmrk/Gallery/Search/SearchBar.vue

```diff
@@ -170,12 +237,18 @@
 </template>
 
 <script lang="ts">
-import { Component, Emit, Prop, Ref, mixins } from 'nuxt-property-decorator'
+import {
```

Comment:
hi @Jarsen136, is it possible to move `Search` components outside `rmrk` folders? maybe move the component to `/components/search/`

Reply:
yes! For future multi-chain components it would be really appreciated :)

Reply:
Done